pints/tests/test_interfaces_stan.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
+#
+# Tests interface with Stan
+#
+# This file is part of PINTS (https://github.com/pints-team/pints/) which is
+# released under the BSD 3-clause license. See accompanying LICENSE.md for
+# copyright notice and full license details.
+#
+import pints
+import unittest
+import numpy as np
+import pints.toy
+
+# Consistent unit testing in Python 2 and 3
+try:
+    unittest.TestCase.assertRaisesRegex
+except AttributeError:
+    unittest.TestCase.assertRaisesRegex = unittest.TestCase.assertRaisesRegexp
+
+try:
```

Comment:
See above. If there's an "interfaces" module that isn't just for stan, then it should be possible to import it with or without stan

Reply:
Ok, cool. See my comment above about whether that means I need to put the Stan stuff in a separate folder